core/conversion/converters/impl/expand.cpp

```diff
@@ -16,33 +16,6 @@ namespace converters {
 namespace impl {
 namespace {
 
-nvinfer1::ILayer* create_plugin(
-    ConversionCtx* ctx,
-    const torch::jit::Node* n,
-    nvinfer1::ITensor* inShape,
-    nvinfer1::ITensor* expandShape,
-    int32_t in_rank,
-    int32_t expand_rank,
-    const char* name) {
-  auto creator = new plugins::CheckShapePluginCreator();
-  std::vector<nvinfer1::PluginField> fields;
-  nvinfer1::PluginField input_rank("input_rank", &in_rank, nvinfer1::PluginFieldType::kINT32, 1);
-  nvinfer1::PluginField output_rank("expand_rank", &expand_rank, nvinfer1::PluginFieldType::kINT32, 1);
-  fields.push_back(input_rank);
-  fields.push_back(output_rank);
-  nvinfer1::PluginFieldCollection collection;
-  collection.nbFields = fields.size();
-  collection.fields = fields.data();
-  auto plugin = creator->createPlugin(name, &collection);
-
-  nvinfer1::ITensor* inputs[] = {inShape, expandShape};
-  auto expandShape_layer = ctx->net->addPluginV2(inputs, 2, *plugin);
-  TRTORCH_CHECK(expandShape_layer, "Unable to create interpolation plugin from node" << *n);
-
-  expandShape_layer->setName("CheckShapePlugin");
-  return expandShape_layer;
-}
-
 void addSliceInput(nvinfer1::Dims& dims, int idx, ConversionCtx* ctx, nvinfer1::ISliceLayer* slice) {

```

Comment:
Same as previous comments. We probably don't need this function. This can be replaced using tensor_to_const method [https://github.com/NVIDIA/TRTorch/blob/master/core/conversion/converters/Weights.h#L25](url). We can directly call this in the add_expand_dynamic itself maybe ?

Reply:
Thanks for your advice. I replace this function with tensor_to_const.